test/libyul/yulOptimizerTests/fullSimplify/operations.yul

```diff
@@ -0,0 +1,44 @@
+{
+  let x := mload(0)
+  mstore(1, mul(x, 0))
+  mstore(2, div(x, 0))
+  mstore(3, div(0, x))
+  mstore(4, sdiv(x, 0))
+  mstore(5, sdiv(0, x))
+  mstore(6, and(0, x))
+  mstore(7, and(x, 0))
+  mstore(8, mod(0, x))
+  mstore(9, mod(x, 0))
+  mstore(10, lt(x, x))
+  mstore(11, gt(x, x))
+  mstore(12, slt(x, x))
+  mstore(13, sgt(x, x))
+  mstore(14, mod(x, x))
+  mstore(15, and(x, not(x)))
+  mstore(16, and(not(x), x))
+  mstore(17, or(x, not(x)))
+  mstore(18, or(not(x), x))
+}
+// ----
+// fullSimplify
```

Comment:
What is this format btw?

Reply:
The string here is the optimization step to apply. `isoltest` ensures that it matches the directory name. I could have used the directory name only, but then you would not know which kind of optimization you are looking at by just looking at the file contents.